### ISSUE TITLE
Select frame for thumbnail extraction

### ISSUE THREAD

**Issue by Dreamink-Official** [ynput/ayon-core#1051]
### Is there an existing issue for this?

- [x] I have searched the existing issues.

### Please describe the feature you have in mind and explain what the current shortcomings are?

Right now when using the extract_thumbnail plugin AYON is picking the middle frame by default.
It would be nice to have an option in the settings to select a specific frame which is used for that.

### How would you imagine the implementation of the feature?

 ![Image](https://github.com/user-attachments/assets/18775457-de53-44c5-8183-8a26060c3a90)

- Add a optional field to type in the frame which should be selected for the thumbnail
- When no frame is selected it should use the middle frame by default

### Are there any labels you wish to add?

- [x] I have added the relevant labels to the enhancement request.

### Additional context:

I have already implemented that locally but a question about the field in the AYON publisher: How are you handling frame ranges in AYON? For example in my DCC i am starting with frame 1001. But the extraction uses the video file so there the frame count starts by 0. 

In my current implementation for that I prefere to select the frame which is displayed in my dcc so frame 1005 for example.

**Comment by jakubjezek001** [comment]
Hey @Dreamink-Official thank you for the request. I was wondering if it is necessary to know exact number of the frame for thumbnail or whether it could be just explicit frame agnostic approach which is already implemented now. 

Here in settings you are able to set float number referring to what part of total 100% range the frame should be taken. Yes it is approximating a little. 

`ayon+settings://core/publish/ExtractThumbnail/duration_split`

**Comment by Dreamink-Official** [comment]
For me, (as a developer), the explicit frame-agnostic approach would be fine. But my artists specifically asked for a way to select the frame shown as the thumbnail, so I added this feature.

They told me:
- Sometimes the selected frame just doesn’t represent the shot well. Being able to pick a specific frame lets them show the most important moment.
- It gives them more control over how their work is presented, which matters a lot to them.
- It’s just a bit more flexible. Most of the time, the frame-agnostic approach works fine, but when you really need to highlight something specific, this makes it possible.

**Comment by BigRoy** [comment]
> But my artists specifically asked for a way to select the frame shown as the thumbnail, so I added this feature.

Any chance you can contribute back (or can share) your changes?

**Comment by Dreamink-Official** [comment]
Of course! I did created a PR back in time for that (https://github.com/ynput/ayon-core/pull/1052)
